src/main/flight/pid.c

```diff
@@ -723,6 +783,12 @@ static void pidRewrite(pidProfile_t *pidProfile, controlRateConfig_t *controlRat
         // -----calculate total PID output
         axisPID[axis] = PTerm + ITerm + DTerm;
```

Comment:
Have no idea. Have never touched anything here. It's there from the very beginning.